include-refactoring/xsimd/types/xsimd_batch.hpp

```diff
@@ -180,6 +181,11 @@ struct batch_bool : types::get_bool_simd_register_t<T, A> {
   template<size_t... Is>
   batch_bool(bool const* data, detail::index_sequence<Is...>);
 
+  template <class U, class... V, size_t I, size_t... Is>
+  static register_type make_register(detail::index_sequence<I, Is...>, U u, V... v);
```

Comment:
Why not keeping the base type as a POD type? It would make this kind of function useless

Reply:
Not sur to get you. The base type is still a POD type. The reason for this function is that I wante to instantiate the register type with a single bool value. This requires either a `kernel::broadcast` function for batch bool, or repeating the bool N times so that we can do `register{arg...}`.